tests/storage/glusterfs/conftest.py

```diff
@@ -88,22 +88,15 @@ def pool_with_glusterfs(host):
 @pytest.fixture(scope='session')
 def gluster_disk(host, sr_disk_for_all_hosts):
     sr_disk = sr_disk_for_all_hosts
-    device = '/dev/' + sr_disk
     hosts = host.pool.hosts
+    mountpoint = '/mnt/sr_disk'
     for h in hosts:
         logging.info(">> Format sr_disk %s and mount it on host %s" % (sr_disk, h))
-        h.ssh(['mkfs.xfs', '-f', device])
-        h.ssh(['rm', '-rf', '/mnt/sr_disk']) # Remove any existing leftover to ensure rmdir will not fail in teardown
-        h.ssh(['mkdir', '-p', '/mnt/sr_disk'])
-        h.ssh(['cp', '-f', '/etc/fstab', '/etc/fstab.orig'])
-        h.ssh(['echo', '%s /mnt/sr_disk xfs defaults 0 0' % device, '>>/etc/fstab'])
-        h.ssh(['mount', '/mnt/sr_disk'])
+        setup_formatted_and_mounted_disk(h, sr_disk, 'xfs', mountpoint)
     yield
     for h in hosts:
         logging.info("<< Restore fstab and unmount /mnt/sr_disk on host %s" % h)
-        h.ssh(['cp', '-f', '/etc/fstab.orig', '/etc/fstab'])
-        h.ssh(['umount', '/mnt/sr_disk'])
-        h.ssh(['rmdir', '/mnt/sr_disk'])
+        teardown_formatted_and_mounted_disk(h, mountpoint)
```

Comment:
Would this log be relevant in the called fonctions?

Reply:
i don't know, but it doesn't seem to hurt, so i just replace the exact lines and not that one.

Reply:
Second answer to "why": because it is not related to the objective of your commit. We don't want to mix unrelated changes together. Formatting changes that you want to do to would belong to a separate commit (and a different PR usually), if relevant.

Reply:
I looked at the code. Logs are already in the functions so you can remove the logs as well :)